libraries/lib-time-and-pitch/tests/StaffPadTimeAndPitchTest.cpp

```diff
@@ -0,0 +1,116 @@
+#include "StaffPadTimeAndPitch.h"
+#include "AudioContainer.h"
+#include "WavFileIO.h"
+
+#include <catch2/catch.hpp>
+
+namespace fs = std::filesystem;
```

Comment:
We can't use std::filesystem with are minimum macOS target, and we won't increase it just for these unit tests.  Please rewrite the tests without it.